tailwind.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
       },
       colors: {
         darkgreen: "#00672E",
+        primary: "#EFECEA",
```

Comment:
Change custom class to "primarybg" to be more specific.
```suggestion
        primarybg: "#EFECEA",
```